licenses/README.rst

```diff
@@ -1,5 +1,6 @@
 Licenses
 ========
 
-This directory holds license and credit information for astropy, works astropy is derived from, and/or datasets.
+This directory holds license and credit information for works astropy is derived from or distributes, and/or datasets.
 
+License file for the astropy itself is placed in the root directory of this repository.
```

Comment:
Can you add 'The' at the start here and add 'package' after 'astropy'? It should be 'The license file for the astropy package is'